angular-ngrx-scss/projects/prism/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "prism",
   "version": "0.0.1",
   "peerDependencies": {
-    "@angular/common": "^13.0.0",
-    "@angular/core": "^13.0.0"
+    "@angular/common": "^16.1.0",
```

Comment:
Can we lock these to the specific version as well? I always forget this package is in here and I don't know if there's reasons it may need to be a little more flexible. I think the main package.json being locked is more important than this one.